tests/test_mesh.py

```diff
@@ -269,26 +269,26 @@ def arithmetic_mesh_vector_setup(self):
                                   structured = True)
         self.volumes1 = list(self.mesh_1_vector.structured_iterate_hex("xyz"))
        
-        fracs_vector_tag = self.mesh_1_vector.mesh.tag_get_handle("cell_fracs", 
+        test_vector_tag = self.mesh_1_vector.mesh.tag_get_handle("testing", 
```

Comment:
Let's make this tag name a variable as well as an extra measure of consistency, possibly even at global scope in this test file...

Reply:
Even better is to make it a member of the testing class that is used here

Reply:
Okay. So when I first assign data to the variable, just add the `global` scope indicator? I wouldn't need to do it anywhere else then.

Reply:
So just self.test_vector_tag = ...

Reply:
I think this went off in the wrong direction.  I was thinking that you'd define something like

`vector_math_tag_name = "testing"`

and then use that in place of providing the string literal `"testing"`

Reply:
Oh, I completely misunderstood. I am on the same page now.